docs/standards/rest.md

```diff
@@ -115,6 +115,46 @@ headers:
 
 [Header field names are case insensitive](https://datatracker.ietf.org/doc/html/rfc7230#section-3.2). Snyk v3 API specs must use kebab case for consistency. All non-standard headers that are unique to Snyk must begin with `snyk-` (e.g. `snyk-requested-version`).
 
+## <a id="request-parameters"></a>Request Parameters
+
+### <a id="formats"></a>Formats
+
+Some resource types may be expressed with a media content-type format other than JSON API. These alternative formats may be offered with these format request parameters:
```

Comment:
I'd expect this header to be optional, and that all resources must have a JSON API representation.

Reply:
I think the initial reason that this came up, was that for SBOM documents it does not make sense to have a JSON API representation. It’s certainly possible, but @garethr brought up the point that it gives no added value to the consumer. Responses need to be parsed and queried to retrieve the actual content, the JSON API envelop is just a necessary wrapper to get it past validation, but other than that none of the added data from JSON API yield any benefit to the consumer; in fact it makes usage by the consumers slightly more cumbersome. @garethr does this somewhat sum up your concerns?

@ianlivingstone are you concerned that we introduce a backdoor for teams at Snyk to implement services that don’t adhere to JSON API, when in fact they should?

Reply:
@ianlivingstone "may be expressed" / "may be offered" in the https://datatracker.ietf.org/doc/html/rfc2119#section-5 sense.

@mcombuechen JSON API enables our consumers to paginate through collections, and navigate a graph of resources through the relationships they declare. It's not really for me to say whether it makes sense to represent SBOM content in some abstract representation in JSON API attributes -- this is certainly a domain-specific judgement call.

If it does not make sense to represent an SBOM _content_ as JSON API -- we should still provide a JSON API response which at a minimum:
- Identifies the resource (`id` and `type`)
- Relates to the supported media type representations of the content, as relationship links.

Here's why:
- Supports aggregating a bulk collection of SBOMs though relationships (give me all the SBOMS for all the projects in org `org_id`) and pagination.
- Supports async API use cases -- imagine a case where producing an SBOM takes a long time. The JSON API response can indicate the operational state of a request for an SBOM, and a link to the created media content type(s) when it's ready.

Reply:
Updated taking the above into account:

- A JSON API response is a MUST, with some bare minimum content requirements.
- A complete JSON API representation is a SHOULD, but if it doesn't fit or make sense, just implement the minimum.

I think that covers the above, PTAL.